assets/css/main.css

```diff
@@ -293,29 +293,38 @@ html, body {
 
```

Comment:
I just ran this locally on my machine, and is this happening to you as well, or is just me?

![image](https://user-images.githubusercontent.com/43036333/88372266-3707b400-cd9e-11ea-9444-eabd1c7f298e.png)